app/experimenter/experiments/tests/test_views.py

```diff
@@ -286,6 +289,23 @@ def test_filters_experiments_with_surveys(self):
 
         self.assertEqual(set(filter.qs), set([exp_1, exp_2]))
 
+    def test_filters_for_subscribed_experiments(self):
+        exp_1 = ExperimentFactory.create(name="Experiment", slug="experiment")
+        ExperimentFactory.create()
+        ExperimentFactory.create()
+
+        exp_1.subscribers.add(self.user)
+
+        self.assertTrue(self.user in exp_1.subscribers.all())
```

Comment:
I don't think this assertion is useful, since it is mostly testing that the ORM works as documented.

Reply:
removed!